src/core_plugins/kibana/public/context/api/__tests__/anchor.js

```diff
@@ -0,0 +1,80 @@
+import expect from 'expect.js';
+import sinon from 'sinon';
+
+import { fetchAnchor } from 'plugins/kibana/context/api/anchor';
+
+
```

Comment:
I've been using the following rules:

* Group third-party and local imports using single blank lines, sort alphabetically by filename within the groups.
* Separate the import section from other code by two blank lines.
* Separate the export section from other code by two blank lines.

This results in:

```
// third-party/global imports
// ...

// local imports
// ...


// code
// ...
// ...


// exports
// ...
```